packages/synapse-react-client/src/synapse-queries/chat/useChat.ts

```diff
@@ -51,32 +49,33 @@ export function useCreateAgentSession(
   })
 }
 
-export function useGetAgentChatWithAsyncStatus<
-  TData = AsynchronousJobStatus<AgentChatRequest, AgentChatResponse>,
->(
-  request: AgentChatRequest,
-  options?: Partial<
-    UseQueryOptions<
-      AsynchronousJobStatus<AgentChatRequest, AgentChatResponse>,
-      SynapseClientError,
-      TData
-    >
+export function useGetAgentChatWithAsyncStatus(
```

Comment:
Super nitpicky, consider renaming to `useSendChatMessageToAgent` to make the purpose of this mutation more obvious.